graph.py

```diff
@@ -0,0 +1,55 @@
+from collections import defaultdict
+
+class Graph:
+    def __init__(self) -> None:
+        self.adjacency_list = defaultdict(list)
+    
+    def add_edge(self, u, v):
+        self.adjacency_list[u].append(v)
+        self.adjacency_list[v].append(u)
+        
+    def dfs(self, source):
+        stack = [source]
+        visited = set()
+        traversal = []
+        while stack:
+            vertex = stack.pop()
+            if vertex not in visited:
+                visited.add(vertex)
+                traversal.append(vertex)
+                for neighbour in reversed(self.adjacency_list[vertex]):
+                    if neighbour not in visited:
+                        stack.append(neighbour)
+        return traversal
+    
+    def bfs(self, source):
```

Comment:
Method name is 'bfs' ie 'breadth first search' where is the searching parameter? Is this search or traversal?

Reply:
it should be bfs_travesral, I'll change